### ISSUE TITLE
API to matplotlib

### ISSUE THREAD

**Issue by dixiyao** [SabrinaSun1225/ChromaPalette#1]
Is there any intension in future extension of APIs to Matplotlib?

**Comment by SabrinaSun1225** [comment]
Thank you for your question! 
Matplotlib is an excellent Python package, and Seaborn might also satisfy most people's needs. Both of them allow you to customize the color palette using a list of hex codes, which is exactly what my Python package provides.

**Comment by 1sadiqhussain** [comment]
Certainly! Matplotlib is a powerful and widely used library for creating visualizations in Python. As for future extensions of APIs in Matplotlib, it's important to note that the library has a strong developer community and is constantly evolving to meet the needs of users.
